tests/multi_ptr/multi_ptr_common.h

```diff
@@ -11,9 +11,70 @@
 
 #include "../common/common.h"
 #include "../common/type_coverage.h"
+#include "../common/type_list.h"
+
+namespace Catch {
```

Comment:
I see this code in the SYCL-2020 branch already. Probably we need update this branch to see the real diff.

Reply:
Merged SYCL-2020 into PR branch, so there is no more changes in `multi_ptr_common.h`